packages/shared/src/contexts/BootProvider.tsx

```diff
@@ -129,8 +130,7 @@ export const BootDataProvider = ({
     );
   };
 
-  const [initialLoad, setInitialLoad] = useState<boolean>(null);
-  const [cachedBootData, setCachedBootData] = useState<Partial<Boot>>(() => {
+  const initialData = useMemo(() => {
```

Comment:
I think we don't have to make this a useMemo since we only need it at the beginning to hold some data for initial render.